ReactiveCocoa/Swift/Scheduler.swift

```diff
@@ -9,44 +9,48 @@
 import Foundation
 
 /// Represents a serial queue of work items.
-public protocol SchedulerType {
+public protocol SchedulerProtocol {
 	/// Enqueues an action on the scheduler.
 	///
 	/// When the work is executed depends on the scheduler in use.
 	///
 	/// Optionally returns a disposable that can be used to cancel the work
 	/// before it begins.
-	func schedule(action: () -> Void) -> Disposable?
+	@discardableResult
```

Comment:
Hmm I think this shouldn't be discardable, it would help us catch errors where we didn't handle the `Disposable`.